data/tests/tech/aria/aria-atomic.json

```diff
@@ -0,0 +1,577 @@
+{
+  "title": "aria-atomic test",
+  "description": "A suite of tests for aria-atomic",
+  "html_file": "aria/aria-atomic.html",
+  "assertions": [
+    {
+      "feature_id": "aria/aria-atomic_attribute",
+      "feature_assertion_id": "convey_value_false"
+    },
+    {
+      "feature_id": "aria/aria-atomic_attribute",
+      "feature_assertion_id": "convey_value_true"
+    }
+  ],
+  "commands": {
+    "jaws": {
+      "chrome": [
+        {
+          "command": "activate_button",
+          "css_target": "#trigger-1",
+          "before": {
+            "mode": "auto",
+            "focus_location": "target",
+            "virtual_location": "target"
+          },
+          "after": "target",
+          "output": "Announce only what has changed",
```

Comment:
Can you quote actual output so that it is something like`"output": "\"Announce only what has changed\""`? Sometimes the output is augmented with inline notes, so quoting the actual output can help to make it clear what was actually announced.